server/routes/index.js

```diff
@@ -1,8 +1,22 @@
 var express = require("express");
 var router = express.Router();
+const auth = require('../controllers/authController.api.js');
+const { verifyJwt } = require('../middlewares');
 
-router.get("/welcome", function(req, res, next) {
+router.get("/welcome", function (req, res, next) {
   res.status(200).send({ welcomeMessage: "Step 1 (completed)" });
 });
 
+router.post('/register', auth.register);
+router.post('/login', auth.login);
+
```

Comment:
For these I think we can make a separate files for our routes. These can be called auth.js or something like that and then import it into index.js as authRoutes or something. So it would just be router.use('/', authRoutes) here. I did the same with my projects routes.

Reply:
I agree. Same can be done with protected routes.